debezium-testing/tmt/tests/debezium/oracle-init/logminer-init-pdb.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TABLESPACE logminer_tbs DATAFILE '/opt/oracle/oradata/FREE/FREEPDB1/logminer_tbs.dbf' SIZE 25M REUSE AUTOEXTEND ON MAXSIZE UNLIMITED;
```

Comment:
I would suggest renaming this file to indicate this is explicitly for Oracle FREE since we hard-code the DB and PDB names.